src/app/community-view/community-view.component.spec.ts

```diff
@@ -229,7 +263,9 @@ describe('CommunityViewComponent', () => {
     expect(component.canLeaveCommunity).toBeTruthy();
     component.leaveCommunity();
     fixture.detectChanges();
-    expect(component.isCommunityMember).toBeFalsy();
+    const matDialog: MatDialog = TestBed.get(MatDialog);
```

Comment:
With these changes in mind I suggest to rename the test to `should open a dialog to confirm user leaves a community` because it actually checks if the dialog is opened and not if user left the community.

Reply:
thank you. done